tsl/src/continuous_aggs/invalidation.c

```diff
@@ -361,7 +361,7 @@ cut_invalidation_along_refresh_window(const CaggInvalidationState *state,
 		/*
 		 * Entry completely enclosed so can be deleted:
 		 *
-		 * |---------------|
+		 * [---------------)
```

Comment:
FWIW, I used the different style to denote the difference between refresh window and invalidation. Don't mind changing though if you prefer.

Reply:
The original style (which is yours, right) doesn't allow to see which side is open or close. This led into error in a diagram in another comment in this file.